itl/plugins-contrib.d/raid-controller.conf

```diff
@@ -31,16 +31,92 @@ object CheckCommand "lsi-raid" {
 
 	arguments = {
 		"-C" = {
-			required = true
 			value = "$lsi_controller_number$"
 			description = "Insert the controller number to be checked."
 		}
 		"-p" = {
-			required = true
```

Comment:
Why are those arguments removed as requirement?

Reply:
Responded in next comment.